src/pages/dashboard/Dashboard.tsx

```diff
@@ -173,6 +173,8 @@ const Dashboard = () => {
           </StyledLinkContent>
         </StyledSearchButton>
         <StyledNewRegistrationButton
+          as={Link}
+          to={user ? UrlPathTemplate.NewRegistration : UrlPathTemplate.Login}
           onClick={() => {
```

Comment:
onCLicken kan kanskje fjernes nå? eller trigges den selv om det er link?

(hvis den ikke trigges må brukeren trykke på knappen 2 ganger for å komme til ny registrering om man ikke er innlogget fra før da)

Reply:
den trigges allikevel ja. Må nok ha den for å sette localstorage-keyen riktig etter login!

Reply:
men kan fjerne noe av det ja!

Reply:
nice, litt forvirrende at `to` og `onClick` ser ut til å gjøre nøyaktig det samme, men må sikkert bare bli sånn da :p